pkg/role_binding/role_binding.go

```diff
@@ -119,10 +128,32 @@ func PresentRoleBinding(rb *v1.RoleBinding) RoleBinding {
 	}
 }
 
-func PresentRoleBindings(roleBindings []*v1.RoleBinding) []string {
-	var roleList []string
+func PresentRoleBindings(roleBindings []*v1.RoleBinding) []AccountRoleBinding {
+	keySeparator := "__"
+	uniqueRoleBindingsMap := make(map[string]int)
+	var uniqueRoleList []AccountRoleBinding
 	for _, roleBinding := range roleBindings {
-		roleList = append(roleList, roleBinding.Role().ID())
+		if val, ok := uniqueRoleBindingsMap[roleBinding.Role().ID()+keySeparator+roleBinding.Type()]; ok {
+			uniqueRoleBindingsMap[roleBinding.Role().ID()+keySeparator+roleBinding.Type()] = val + 1
+		} else {
+			uniqueRoleBindingsMap[roleBinding.Role().ID()+keySeparator+roleBinding.Type()] = 1
```

Comment:
consider assigning roleBinding.Role().ID()+keySeparator+roleBinding.Type() to a variable and reusing it for clarity